src/frontend/src/pages/FinancePage/ReimbursementRequestPage/ReimbursementRequestDetailsView.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Chip, Grid, Paper, Table, TableBody, TableCell, TableHead, TableRow, Typography, useTheme } from '@mui/material';
+import { Box } from '@mui/system';
+import { ReimbursementProduct, ReimbursementRequest, wbsPipe } from 'shared';
+import { datePipe, fullNamePipe } from '../../../utils/pipes';
+import VerticalDetailDisplay from '../../../components/VerticalDetailDisplay';
+import PageLayout from '../../../components/PageLayout';
+
+interface ReimbursementRequestDetailsViewProps {
+  reimbursementRequest: ReimbursementRequest;
+}
+
+const ReimbursementRequestDetailsView: React.FC<ReimbursementRequestDetailsViewProps> = ({ reimbursementRequest }) => {
+  const theme = useTheme();
+  const totalCostBackgroundColor = theme.palette.mode === 'dark' ? theme.palette.grey[800] : theme.palette.grey[200];
+
+  const BasicInformationView = () => {
+    return (
+      <>
+        <Typography variant="h6">Basic Information</Typography>
+        <Grid container spacing={2}>
+          <Grid item sm={6} xs={12}>
+            <VerticalDetailDisplay label="Purchased From" content={reimbursementRequest.vendor.name} />
+          </Grid>
+          <Grid item sm={6} xs={12}>
+            <VerticalDetailDisplay label="Sabo Number" content={`${reimbursementRequest.saboId ?? '----'}`} />
+          </Grid>
+          <Grid item sm={6} xs={12}>
+            <VerticalDetailDisplay label="Refund Source" content={`${reimbursementRequest.account}`} />
+          </Grid>
+          <Grid item sm={6} xs={12}>
+            <VerticalDetailDisplay label="Expense Type" content={`${reimbursementRequest.expenseType.name}`} />
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            container
+            mt={2}
+            ml={2}
+            sx={{ backgroundColor: totalCostBackgroundColor, borderRadius: '10px' }}
+          >
+            <Grid item xs={6} textAlign={'center'}>
+              <Typography fontSize={50}>Total Cost</Typography>
+            </Grid>
+            <Grid xs={6} sx={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+              <Typography fontSize={50}>{`$${reimbursementRequest.totalCost}`}</Typography>
+            </Grid>
+          </Grid>
+        </Grid>
+      </>
+    );
+  };
+
+  const ReimbursementProductsView = () => {
```

Comment:
this is complicated enough to be in it's own file imo,
also I think it should be bigger/ take up more of this dead space around it
<img width="394" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/56bf2e36-2e7a-48d1-98c8-e099e33aac60">

Reply:
oop i did math wrong